core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java

```diff
@@ -323,6 +343,24 @@ private void tryToMakeAdmin(User u) {
      *      a valid {@link User} object if the user creation was successful.
      */
     private User createAccount(StaplerRequest req, StaplerResponse rsp, boolean selfRegistration, String formView) throws ServletException, IOException {
+        SignupInfo si = validateAccountCreationForm(req, selfRegistration);
+
+        if(si.errorMessage!=null) {
```

Comment:
I suggest you to follow the same code style: `if (si.errorMessage != null) {`

Reply:
I was leaving the existing code alone because it said in one of the many "how to contribute" documents not to include unrelated formatting changes and I guess I was overdoing it :) Will change.

Reply:
The idea is to modify only what you need. But, if you modify a line where you can apply a code style (according to the rest of the file), my suggestion is to do it.